### ISSUE TITLE
World browser maximum amount of items is set to 52 instead of 55

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1424]
### Describe the bug?

The world browser can only display 52 items at a time which is not enough to fill the full page it might depend on aspect ratio

### To Reproduce

Open the world browser go to published worlds

### Expected behavior

The world browser should be able to display 55 items before going to the next page

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/70e241d7-9322-4744-9926-7c958b39fe87)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/cab6817c-2a0d-489c-b6d3-20a8cfdf5834)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/2a63877a-65bd-4212-80f1-3a6212553a4f)
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/53fdce8f-f231-4a40-bf64-6869d9fa2e91)

### Resonite Version Number

2024.2.12.1430

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

I don't believe a log file is irrelevant to this issue but can be provided if asked

### Reporters

@epicEaston197 @Nytra

**Comment by Nytra** [comment]
I think the issue here is that the number of items per row is different is desktop mode vs VR. In VR it's 4, but in desktop mode it's 5. 

The default MaxItems (52) is a multiple of 4 so works correctly in VR, but not in desktop mode.

**Comment by Muppeq** [comment]
So using 40 or 60 would fill the rows in VR and Desktop.

**Comment by Banane9** [comment]
Yup, gotta use a compound multiple of 4 and 5 to work evenly with both.

**Comment by Frooxius** [comment]
Changed this to 60 in 2024.3.1.1178, thanks!

**Comment by Nytra** [comment]
I think this fix has reverted somehow, MaxItems in my world list is set to 52 again

**Comment by shiftyscales** [comment]
Seems this regressed at some point, @Frooxius.
